src/proofOfConcept.java

```diff
@@ -0,0 +1,106 @@
+
+import components.map.Map;
+import components.map.Map1L;
+import components.sequence.Sequence;
+import components.sequence.Sequence1L;
+
+public final class proofOfConcept {
```

Comment:
I am somewhat skeptical that this runs. Traditionally, Java doesn't let you have more than one class in a file unless they're nested. Newer versions of Java may allow this, but I'm not sure how they handle the issue of file names needing to match the class name. Also, class names should start with a capital letter. 
```suggestion
public final class ProofOfConcept {
```